<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreConfig</class>
 <widget class="QDialog" name="PreConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>轴承3名称</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="10" column="3">
       <widget class="QLineEdit" name="txt_DrivenwheelTeethNum"/>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txt_bearing2_name"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>测点类型</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>主动轮名称</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>传感器灵敏度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txt_name"/>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>从动轮齿数</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>通道号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="txt_bearing4_name"/>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>从动轮名称</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>主动轮齿数</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLineEdit" name="txt_DrivenwheelName"/>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cbox_bearing1_model"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_id"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>轴承3型号</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>轴承1型号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_ch"/>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="txt_capstanName"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>轴承2名称</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="txt_ShaftDiameter"/>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>软件版本</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>轴承2型号</string>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>轴承1名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="txt_bearing3_name"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="txt_bearing1_name"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>前置处理器ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>轴承4名称</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QLineEdit" name="txt_version"/>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>是否使能</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="txt_capstanTeethNum"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>轴承4型号</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txt_Sensitivity"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>轴径</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cbox_bearing3_model"/>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="cbox_bearing4_model"/>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="cbox_bearing2_model"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
